src/App.jsx

```diff
@@ -3,26 +3,64 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getAllTodos, getUser } from './api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getAllTodos().then(data => this.setState({
+      todos: data,
+    }));
+  }
+
+  selectUser = (id) => {
+    this.setState({
+      selectedUserId: id,
+    });
+  }
+
+  clearUser = () => {
+    this.setState({
+      selectedUserId: 0,
+    });
+  }
+
+  shuffleTodos = () => {
+    const { todos } = this.state;
+    const shuffledTodos = [...todos].sort(() => 0.5 - Math.random());
```

Comment:
Warning!
The use of this algorithm is not recommended, because it is inefficient and strongly biased; see comments. It is being left here for future reference, because the idea is not that rare.

Почитай за крутий [shuffle](https://stackoverflow.com/a/12646864) масива